test/integration_test/common_test.go

```diff
@@ -51,17 +52,18 @@ import (
 )
 
 const (
-	nodeDriverName        = "ssh"
-	schedulerDriverName   = "k8s"
-	remotePairName        = "remoteclusterpair"
-	remoteConfig          = "remoteconfigmap"
-	specDir               = "./specs"
-	defaultClusterPairDir = "cluster-pair"
-	pairFileName          = "cluster-pair.yaml"
-	remoteFilePath        = "/tmp/kubeconfig"
-	configMapSyncWaitTime = 3 * time.Second
-	defaultSchedulerName  = "default-scheduler"
-	bucketPrefix          = "stork-test"
+	nodeDriverName               = "ssh"
+	schedulerDriverName          = "k8s"
+	remotePairName               = "remoteclusterpair"
+	remoteConfig                 = "remoteconfigmap"
+	specDir                      = "./specs"
+	defaultClusterPairDir        = "cluster-pair"
+	pairFileName                 = "cluster-pair.yaml"
+	remoteFilePath               = "/tmp/kubeconfig"
+	configMapSyncWaitTime        = 3 * time.Second
+	defaultSchedulerName         = "default-scheduler"
+	bucketPrefix                 = "stork-test"
+	torpedoObjectStoreDriverName = "torpedo_objectstore"
```

Comment:
Can you call this the default driver instead of torpedo driver?

Reply:
Fixed.